client/src/components/Error/Error.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Error.css'
+
+const Error=()=>{
+    return(
+        <div className="container pt-5 mt-5">
+            <hr />
+            <div className="text-center text-danger firstdiv">
```

Comment:
@EmperorYP7  Have a look. Do classes have consistent naming conventions?
LGTM

Reply:
The uppermost div should be given a `className = "error-component container"`. We wont be using `pt` and `mt` or any such classes provided by bootstrap. Margins and padding should be handled manually in the corresponding CSS.